src/Umbraco.Core/Persistence/SqlSyntax/SqlCeSyntaxProvider.cs

```diff
@@ -163,7 +165,8 @@ public override void WriteLock(IDatabase db, params int[] lockIds)
             if (db.Transaction.IsolationLevel < IsolationLevel.RepeatableRead)
                 throw new InvalidOperationException("A transaction with minimum RepeatableRead isolation level is required.");
 
-            db.Execute(@"SET LOCK_TIMEOUT 1800;");
+            var timeOut = Current.Configs.Global().SqlWriteLockTimeOut;
+            db.Execute(@"SET LOCK_TIMEOUT " + timeOut  + ";");
```

Comment:
@nul800sebastiaan maybe it would be better with SQL parameters here although it should be safe enough that timeout is an `int` here and not vulnerable to SQL injection.

One of these should work as well though, but I haven't tested it :)

```
var sql = new Sql(@"SET LOCK_TIMEOUT @0;", timeOut);
db.Execute(sql);
```

```
var sql = new Sql(@"SET LOCK_TIMEOUT @timeout;", new { timeout = timeOut });
db.Execute(sql);
```

Reply:
Thanks @bjarnef - [since we force it to be an `int`](SET LOCK_TIMEOUT), we don't have to be worried about SQL injection here.

Reply:
Yeah, I noticed that one. It is probably just a matter of readability or preferred use of NPoco.
Alternatively it could also use string interpolation syntax.